install.sh

```diff
@@ -6,37 +6,21 @@ RELEASE="$(rpm -E %fedora)"
 KERNEL_SUFFIX=""
 QUALIFIED_KERNEL="$(rpm -qa | grep -P 'kernel-(|'"$KERNEL_SUFFIX"'-)(\d+\.\d+\.\d+)' | sed -E 's/kernel-(|'"$KERNEL_SUFFIX"'-)//')"
 
-RPMFUSION_MIRROR_RPMS="https://mirrors.rpmfusion.org"
-if [ -n "${RPMFUSION_MIRROR}" ]; then
-    RPMFUSION_MIRROR_RPMS=${RPMFUSION_MIRROR}
-fi
-
-curl -Lo /tmp/rpms/rpmfusion-free-release-"${RELEASE}".noarch.rpm "${RPMFUSION_MIRROR_RPMS}"/free/fedora/rpmfusion-free-release-"${RELEASE}".noarch.rpm
-curl -Lo /tmp/rpms/rpmfusion-nonfree-release-"${RELEASE}".noarch.rpm "${RPMFUSION_MIRROR_RPMS}"/nonfree/fedora/rpmfusion-nonfree-release-"${RELEASE}".noarch.rpm
+RPMFUSION_MIRROR_RPMS="${RPMFUSION_MIRROR:-https://mirrors.rpmfusion.org}"
 
-curl -Lo /etc/yum.repos.d/_copr_ublue-os_staging.repo https://copr.fedorainfracloud.org/coprs/ublue-os/staging/repo/fedora-"${RELEASE}"/ublue-os-staging-fedora-"${RELEASE}".repo
-curl -Lo /etc/yum.repos.d/_copr_kylegospo_oversteer.repo https://copr.fedorainfracloud.org/coprs/kylegospo/oversteer/repo/fedora-"${RELEASE}"/kylegospo-oversteer-fedora-"${RELEASE}".repo
-
-rpm-ostree install \
-    /tmp/rpms/*.rpm \
+declare -a REPO_PACKAGES=(
     fedora-repos-archive
+    "${RPMFUSION_MIRROR_RPMS}/free/fedora/rpmfusion-free-release-${RELEASE}.noarch.rpm"
+    "${RPMFUSION_MIRROR_RPMS}/nonfree/fedora/rpmfusion-nonfree-release-${RELEASE}.noarch.rpm"
+)
 
-# Handle Kernel Skew with override replace
-rpm-ostree cliwrap install-to-root /
-if [[ "${KERNEL_VERSION}" == "${QUALIFIED_KERNEL}" ]]; then
-    echo "Installing signed kernel from kernel-cache."
-    cd /tmp/kernel-rpms
-    rpm2cpio /tmp/kernel-rpms/kernel-core-*.rpm | cpio -idmv
-    cp ./lib/modules/*/vmlinuz /usr/lib/modules/*/vmlinuz
-    cd /
-else
-    echo "Install kernel version ${KERNEL_VERSION} from kernel-cache."
-    rpm-ostree override replace \
-        --experimental \
-        /tmp/kernel-rpms/kernel-[0-9]*.rpm \
-        /tmp/kernel-rpms/kernel-core-*.rpm \
-        /tmp/kernel-rpms/kernel-modules-*.rpm
-fi
+curl -fSsLo /etc/yum.repos.d/_copr_ublue-os_staging.repo https://copr.fedorainfracloud.org/coprs/ublue-os/staging/repo/fedora-"${RELEASE}"/ublue-os-staging-fedora-"${RELEASE}".repo
+curl -fSsLo /etc/yum.repos.d/_copr_kylegospo_oversteer.repo https://copr.fedorainfracloud.org/coprs/kylegospo/oversteer/repo/fedora-"${RELEASE}"/kylegospo-oversteer-fedora-"${RELEASE}".repo
+
+{
+    printf "%s\0" "${REPO_PACKAGES[@]}"
+    find "$RPMS_DIR"/{config,akmods} -type f -name "*.rpm" -print0
+} | xargs -0 rpm-ostree install
```

Comment:
It takes effort to read the new array+xargs version of this code for installing extra repos vs the previous version being a simple multi-line install command.

I'd suggest something like:
```
rpm-ostree install \
    fedora-repos-archive \
    "$RPMS_DIR"/akmods/*.rpm \
    "$RPMS_DIR"/config/*.rpm \
    "${RPMFUSION_MIRROR_RPMS}/free/fedora/rpmfusion-free-release-${RELEASE}.noarch.rpm" \
    "${RPMFUSION_MIRROR_RPMS}/nonfree/fedora/rpmfusion-nonfree-release-${RELEASE}.noarch.rpm"
```

Reply:
While i agree with the readability point, i strongly object the usage of shell glob expansion for selecting files to be passed to commands.  

It might actually be okay in this case, but also introduces non-obviuos pitfalls:
- limited argument count (most likely irrelevant)
- ignoring dot-prefixed files (most likely irrelevant)
- ignoring files in subdirectories (could lead to confusion)

In my opinion passing a unknown, variable number of (also unknown and variable) file paths as arguments should always be done in a safe way. I would not consider shell glob expansion without a filename prefix && suffix to be safe enough.

To keep readability, i'd like to propose a solution relying on two `rpm-ostree install` calls instead:

```
rpm-ostree install \
    fedora-repos-archive \
    "${RPMFUSION_MIRROR_RPMS}/free/fedora/rpmfusion-free-release-${RELEASE}.noarch.rpm" \
    "${RPMFUSION_MIRROR_RPMS}/nonfree/fedora/rpmfusion-nonfree-release-${RELEASE}.noarch.rpm"
find "$RPMS_DIR"/{config,akmods} -type f -name "*.rpm" -print0 | xargs -0 rpm-ostree install
```

Reply:
In this case, we don't actually want to install everything from akmods. We only want to install the package ublue-os-akmods-addons and exclude everythin in the kmods subdirectory. Similarly to config, the desired rpms are all in the rpm directory.

Reply:
> In this case, we don't actually want to install everything from akmods.
Yeah, i saw that. It is the `ublue-os` subdirectory which is mounted to `akmods`. Mabye that's a little confusing.

Reply:
I updated it to install from `"$RPMS_DIR"/{config,akmods/ublue-os}` and mount `/rpms` instead of `/rpms/ublue-os`.  
This should be less confusing.  
f559aaef127c68b8bf24ab7c8ffb55c384253572